Orange/widgets/visualize/owvenndiagram.py

```diff
@@ -1035,9 +1040,6 @@ def setText(self, text):
             self._text = text
             self.text.setPlainText(text)
 
-    def text(self):
```

Comment:
This function was shadowed by an attribute set in `__init__` and was inaccessible.